assets/js/util.js

```diff
@@ -308,17 +332,21 @@ function callApy(options, endpoint) {
     var requestUrl = window.location.protocol + window.location.hostname +
         window.location.pathname + '?' + $.param(requestOptions.data);
 
+    if(useAjax) {
+        return $.ajax(requestOptions);
```

Comment:
> I am not sure what other parameters count while considering behaviour?

I'm not sure what you mean by this.

My question here is that a few lines below this one, you wrote code to switch to POST dynamically when needed. However, we bypass that protection here.